ppr-ui/src/resources/unitNotes.ts

```diff
@@ -66,7 +66,12 @@ export const UnitNotesInfo: Record<UnitNoteDocTypes, UnitNoteInfoIF> = {
   [UnitNoteDocTypes.CONFIDENTIAL_NOTE]: {
     header: 'Confidential Note',
     dropdownText: 'Confidential Note',
-    fee: FeeSummaryDefaults.NO_FEE
+    fee: FeeSummaryDefaults.NO_FEE,
+    reviewSectionNumber: { // section numbers on Review page
+      attention: 2,
+      certifyInfo: 3,
+      staffPayment: 4
+    }
```

Comment:
Would the incremental wrapper class we made be helpful here, so that this isn't required programatically? 
Or is the layout pattern a bit more involved/complex to rely on that?

Reply:
Yes I can use it,
 I would need to do some refactoring on other components to remove defaults for certify Info and find all the instances that rely on the default.

Reply:
Updated. 
I removed the tests the check for the section number, because I was not able to get the numbers to render in the testing environment.